tutorials/nginx-ingress-gke/index.md

```diff
@@ -0,0 +1,353 @@
+balancer frontend IP address.  Here is a basic flow of the nginx ingress
+solution on GKE.
+
+## Nginx Ingress Controller on GKE
+
+<img src="https://github.com/ameer00/community/blob/master/tutorials/nginx-ingress-gke/Nginx%20Ingress%20on%20GCP%20-%20Fig%2002.png" width="65%">
+
+From the Cloud Shell, let us deploy an Nginx controller and service.  
+  
+	kubectl apply -f ingress-nginx.yaml
+
+	service "ingress-nginx" created
+	deployment "ingress-nginx" created
+
+Wait a few moments while the GCP L4 Load Balancer gets deployed.  Confirm that the ingress-nginx service has been deployed and that you have an external IP address associated with the service (recall we configured the ingress-nginx with ServiceType: LoadBalancer)
+
+	kubectl get service ingress-nginx
+
+	NAME            TYPE           CLUSTER-IP    EXTERNAL-IP      PORT(S)                      AGE
+	ingress-nginx   LoadBalancer   10.7.252.85   35.188.177.209   80:30723/TCP,443:32122/TCP   2m
+
+Also check to see that the Ingress Resource now has an IP address, which is
+different from the Ingress Controller EXTERNAL IP address shown above.  
+  
+	kubectl get ingress  
+
+	NAME               HOSTS     ADDRESS          PORTS     AGE  
+	ingress-resource   *         35.224.254.160   80        12m`
+
+# Test ingress and default backend
+
+We should now be able to access the web application by going to the EXTERNAL-IP
+address of the nginx ingress controller (from the output above).  
+
+![image](https://github.com/ameer00/community/blob/master/tutorials/nginx-ingress-gke/Kuard-ingress.png)
+
+To check if the _default-backend_ service is working properly, access any path (other than
+the default path / defined in the ingress resource) and ensure you receive a 404
+message, for example, 
+
+	http://external-ip-of-ingress-controller]/test
+
+You should get the following message  
+  
+	404 page not found 
+
+# Clean Up
+
+From the Cloud Shell.  
+  
+	kubectl delete -f ingress-resource.yaml
+
+	ingress "demo-ingress" deleted
+
+	kubectl delete -f ingress-nginx.yaml
+
+	service "ingress-nginx" deleted
+	deployment "ingress-nginx" deleted
+
+	kubectl delete -f kuard-app.yaml
+
+	service "kuard" deleted
+	deployment "kuard" deleted
+
+	kubectl delete -f nginx-default-backend.yaml
+
+	service "nginx-default-backend" deleted
+	deployment "nginx-default-backend" deleted
+
+Check no deployments, pods, or ingresses exist on the cluster.  
+  
+	kubectl get deployments  
+
+	No resources found.
+
+	kubectl get pods
+
+	No resources found.
+
+	kubectl get ingress
+
+	No resources found.
+
+Delete the GKE Cluster.  
+  
+	gcloud container clusters delete nginx-tutorial  
+
+	The following clusters will be deleted.
+	 - [nginx-tutorial] in [us-central1-f]
+
+	Do you want to continue (Y/n)?  y
+
+	Deleting cluster nginx-tutorial...done.
+	Deleted [https://container.googleapis.com/v1/projects/ameer-1/zones/us-central1-f/clusters/nginx-tutorial].
+
+To delete the git repo, simply remove the directory.  
```

Comment:
"To delete the git repo, simply remove the directory by running the following command:"